pycbc/workflow/segment.py

```diff
@@ -61,7 +61,7 @@ def save_veto_definer(cp, out_dir, tags=None):
     return veto_def_new_path
 
 
-def get_segments_file(workflow, name, option_name, out_dir):
+def get_segments_file(workflow, name, option_name, out_dir, tags=None):
```

Comment:
At the start of this function you will need the `if tags is None: tags=[]` lines.